ws-reconnect.js

```diff
@@ -0,0 +1,41 @@
+const WebSocket = require('ws')
+const debug = require('debug')('websocket-wrapper')
+const EventEmitter2 = require('eventemitter2')
+const DEFAULT_RECONNECT_INTERVAL = 5000
+
+class WebSocketWrapper extends EventEmitter2 {
+  constructor () {
+    super()
+    this._interval = 5000
```

Comment:
Should this interval be configurable? I.e. an option you can pass in to the plugin?